RecordingApp/app/src/main/java/wycliffeassociates/recordingapp/ProjectManager/SourceAudioTaskFragment.java

```diff
@@ -0,0 +1,128 @@
+package wycliffeassociates.recordingapp.ProjectManager;
+
+import android.app.Activity;
+import android.app.Fragment;
+import android.app.ProgressDialog;
+import android.os.Bundle;
+import android.os.Handler;
+import android.os.Looper;
+
+import com.wycliffeassociates.io.ArchiveOfHolding;
+
+import org.apache.commons.io.FileUtils;
+
+import java.io.File;
+import java.io.IOException;
+
+import wycliffeassociates.recordingapp.Reporting.Logger;
+import wycliffeassociates.recordingapp.Utils;
+
+/**
+ * Created by sarabiaj on 9/21/2016.
+ */
+public class SourceAudioTaskFragment extends Fragment {
+
+    private ProgressDialog mPd;
+    private volatile boolean mInProgress = false;
+
+    public interface SourceAudioExportCallback{
```

Comment:
Forgot to delete this file and DatabaseResyncTaskFragment, thanks! The functionality here has been moved to ExportSourceAudioTask